src/main/java/frc/robot/Constants.java

```diff
@@ -145,7 +145,7 @@ public final class Constants {
     public static final double INDEXER_LOADING_WAIT = 0.5;
     public static final double INDEXER_HUMAN_LOADING_WAIT = 1;
 
-    public static final double INDEXER_SHOOTER_REJECTION_TIME = 0.5;
+    public static final double INDEXER_SHOOTER_REJECTION_TIME = 2; // put back to .5 when using full battery
```

Comment:
Why is this a thing?

Reply:
because the battery was not full
"don't commit temporary changes" blah blah blah i don't care

Reply:
The battery voltage will vary in a match, maybe some sort of system to calculate based on voltage of battery? Another _probably_ better method would be to analyze current of shooter to determine when there is a ball there, as there will an increased amount of current when there is a ball touching it.

Reply:
Oh that's helpful information, I wasn't aware of that 
I'll work on that next time we meet